swift/apple/FirezoneKit/Sources/FirezoneKit/Models/Settings.swift

```diff
@@ -27,12 +27,13 @@ struct Settings: Equatable {
   }
 
   // Convert provider configuration (which may have empty fields if it was tampered with) to Settings
-  static func fromProviderConfiguration(providerConfiguration: [String: String]?) -> Settings {
-    if let providerConfiguration = providerConfiguration {
+  static func fromProviderConfiguration(providerConfiguration: [String: Any]?) -> Settings {
```

Comment:
Why did this change to Any?

Reply:
We take on the responsibility of casting in here instead of making the caller do it